services/frontend-service/src/ui/Releases.tsx

```diff
@@ -159,10 +160,13 @@ const ApplicationBox: React.FC<any> = (props: {
     }
     const releases = application.releases;
     releases?.sort((a, b) => b.version - a.version);
-
+    const warnings = <Warnings name={name} environments={environments} releases={releases} />;
     return (
         <TableRow className="application">
-            <TableCell className="applicationCard">{name}</TableCell>
+            <TableCell className="applicationCard">
```

Comment:
Does it make sense to have a new `TableCell` here with the `Warnings` and the `undeployButton`?

Reply:
done. In the long run, we need a better option though, it looks a bit crowded already.